habitat/tasks/rearrange/multi_task/composite_task.py

```diff
@@ -6,10 +6,10 @@
 
 import copy
 import os.path as osp
-from typing import Any, Dict, List, Optional
+from typing import TYPE_CHECKING, Any, Dict, List, Optional
 
 import numpy as np
-import yaml
+import yaml  # type: ignore[import]
```

Comment:
Good short term solution, but dependency on yaml is completely removed in this PR: https://github.com/facebookresearch/habitat-lab/pull/883